packages/dashboard/src/customization/widgets/queryWidget/multiQueryWidget.tsx

```diff
@@ -23,15 +23,17 @@ const MultiQueryWidgetComponent: React.FC<QueryWidget> = ({ children, ...widget
       accept: ItemTypes.ResourceExplorerAssetProperty,
       drop: ({ queryAssetsParam }: { queryAssetsParam: SiteWiseAssetQuery['assets'] }) => {
         const asset = queryAssetsParam[0];
-
+        // if the widget is an iot-table and is not in experimental mode, we don't want to merge the queries
+        const dontMergeQueries =
+          widget.type === 'iot-table' && !(widget.properties as TableProperties).experimentalMode;
         const updatedWidget = {
           ...widget,
           properties: {
             ...widget.properties,
             queryConfig: {
               ...widget.properties.queryConfig,
               query: {
-                assets: mergeAssetQueries(widget.properties.queryConfig.query?.assets || [], asset),
+                assets: mergeAssetQueries(widget.properties.queryConfig.query?.assets || [], asset, dontMergeQueries),
```

Comment:
can we just return current assets plus new asset here instead of adding this flag?

Reply:
removed.